dotnet/src/SemanticKernel/CoreSkills/HttpSkill.cs

```diff
@@ -83,25 +72,28 @@ public async Task<string> PostAsync(string uri, SKContext context)
     /// <returns>The response body as a string.</returns>
     [SKFunction("Makes a PUT request to a uri")]
     [SKFunctionContextParameter(Name = "body", Description = "The body of the request")]
-    public async Task<string> PutAsync(string uri, SKContext context)
-    {
-        using var httpContent = new StringContent(context["body"]);
-        var response = await this._client.PutAsync(uri, httpContent);
-        var content = response.Content;
-        return await content.ReadAsStringAsync();
-    }
+    public Task<string> PutAsync(string uri, SKContext context) =>
+        this.SendRequestAsync(uri, HttpMethod.Put, new StringContent(context["body"]));
 
     /// <summary>
     /// Sends an HTTP DELETE request to the specified URI and returns the response body as a string.
     /// </summary>
     /// <param name="uri">URI of the request</param>
     /// <returns>The response body as a string.</returns>
     [SKFunction("Makes a DELETE request to a uri")]
-    public async Task<string> DeleteAsync(string uri)
+    public Task<string> DeleteAsync(string uri) =>
```

Comment:
Should we skip reading/returning body content for HttpMethod.Delete?

Reply:
If we don't expect responses, it'd make sense to simplify.  It really depends on the usage you expect; deletes can of course have a response body.

Reply:
Fine to leave it if it's possible. As long as an empty response body won't crash. Thanks!

Reply:
> As long as an empty response body won't crash.

Nope, it'll just result in an empty string.